processing/src/main/java/io/druid/segment/serde/DoubleGenericColumnPartSerdeV2.java

```diff
@@ -36,36 +36,27 @@
 import java.nio.ByteOrder;
 import java.nio.channels.WritableByteChannel;
 
+/**
+ */
 public class DoubleGenericColumnPartSerdeV2 implements ColumnPartSerde
 {
-  private final ByteOrder byteOrder;
-  private Serializer serialize;
-  private final BitmapSerdeFactory bitmapSerdeFactory;
-
   @JsonCreator
   public static DoubleGenericColumnPartSerdeV2 getDoubleGenericColumnPartSerde(
       @JsonProperty("byteOrder") ByteOrder byteOrder,
       @Nullable @JsonProperty("bitmapSerdeFactory") BitmapSerdeFactory bitmapSerdeFactory
   )
   {
-    return new DoubleGenericColumnPartSerdeV2(byteOrder,
-                                              bitmapSerdeFactory != null
-                                              ? bitmapSerdeFactory
-                                              : new BitmapSerde.LegacyBitmapSerdeFactory(), null
+    return new DoubleGenericColumnPartSerdeV2(
+        byteOrder,
+        bitmapSerdeFactory != null
```

Comment:
Code around here is not formatted properly (compare with `DoubleGenericColumnPartSerdeV2`)

Reply:
done.